### ISSUE TITLE
No social-media buttons using firefox on os x.

### ISSUE THREAD

**Issue by maximweb** [gayanvirajith/harmony#13]
Hi Gayan,

first of all: awesome theme! Simple but elegant and all the features I need.

I found a minor glitch but I'm still not sure where this is coming from. When i view your demo at http://gayan.me/harmony I cannot see any of the social media buttons in the footer. I am using Firefox 40.0.3 on an Macbook Pro.

When I visit the very same page via my smartphone all the buttons appear as expected.

I believe this might be an isse in main.css unfortunately i cannot find it. Maybe you can help?

Cheers,
Max

Screenshot Smartphone:
![screenshot_2015-09-17-18-06-55](https://cloud.githubusercontent.com/assets/6138142/9938874/b3441ec0-5d67-11e5-9a6d-09f71ca138fe.png)

Screenshot Firefox:
![harmony-desktop](https://cloud.githubusercontent.com/assets/6138142/9938881/bb7f03c0-5d67-11e5-9809-da56898b72e6.jpg)

**Comment by gayanvirajith** [comment]
Hi Max, 

Thanks for reporting this issue, I will look into this first and get back to you.

Thanks again for using harmony.

**Comment by rpadovani** [comment]
Hi Max,
are you using some tool like PrivacyBadge or AdBlockPlus? 

They hide the icons... try to disable them and visit the page again!

**Comment by maximweb** [comment]
@rpadovani 
Wow, you are absolutely right. I would never have guessed that. Do you know what part of the code exactly triggers the blocking? Since these are no embedded buttons loading code from facebook or google I'd like them to be displayed even with AdBlockPlus activated.

**Comment by rpadovani** [comment]
@maximweb you need to open Filter's preferences (CTRL+ALT+F) and disable *Fanboy's Social Blocking List*

**Comment by maximweb** [comment]
OK, I found the exact issue. It was Fanboy's Social Blocking indeed which blocked by default any block using the css class-name "social-media". My intention to install the blocklist in the firstplace was to avoid the kind of social-media buttons which load external content without even clicking it.

As this is not the case I simply renamed the css class and now it works just fine.

Thanks for your help :+1: and sorry to bother you with that
